hepstats/splot/sweights.py

```diff
@@ -0,0 +1,83 @@
+import numpy as np
```

Comment:
This module should contain a reference to the sPlot paper.

Reply:
It makes sense indeed.